content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -0,0 +1,180 @@
+
+## Workspace-Specific Variables
+
+To view and manage a workspace's variables, go to the workspace and click the **Variables** tab.
+
+The **Variables** page appears, showing all workspace-specific variables and variable sets applied to the workspace. This is where you can add, edit, and delete workspace-specific variables. You can also add and remove variable sets from this workspace.
+
+![Screenshot: The initial appearance of a workspace's variables page](./images/vars.png)
+
+
+### Add a Variable
+
+1. Go to the workspace **Variables** page and click **+ Add variable**.
+
+2. Choose a variable category (Terraform or environment), optionally mark the variable as [sensitive](#sensitive-values), and enter a variable name, value, and optional description.
+
+    Refer to [variable values and format](#variable-values-and-format) for variable limits, allowable values, and formatting.
+
+3. Click **Save variable**. The variable appears in the list of variables for the workspace and Terraform Cloud will begin applying it to runs.
+
+  ![Screenshot: A variable being edited](./images/vars-edit.png)
+
+### Edit a Variable
+
+1. Click the ellipses next to the variable you want to edit and select **Edit**.
+2. Make any desired changes and click **Save variable**.
+
+
+### Delete a Variable
+
+1. Click the ellipses next to the variable you want to delete and select **Delete**.
+2. Type the variable name into the dialog box to confirm and click **Delete**.
+
+## Variable Sets
+
+> **Hands On:** Try the [Create and Use a Variable Set](https://learn.hashicorp.com//tutorials/terraform/cloud-create-variable-set) tutorial on HashiCorp Learn.
+
+To view variable sets for your organization, click **Settings** in the top menu bar and then click **Variable sets** in the left sidebar.
+
+The **Variable sets** page appears, showing a list of all of the variable sets within the organization. Click each variable set to open it and review details about its variables.
+
+![The variable sets page in the Terraform Cloud UI](link)
+
+### Create Variable Sets
+
+-> **Note:** Variable sets are currently in beta.
+
+To create a variable set:
+
+1. Go to the **Variable Sets** page for your organization and click **Create variable set**. The **Create a new Variable set** page appears.
+
+  ![The create a new Variable set page in the Terraform Cloud UI](link)
+
+
+2. Choose a descriptive **Name** for the variable set. You can use any combination of numbers, letters, and characters.
+
+3. Write an optional **Description** that tells other users about the purpose of the variable set and what it contains.
+
+4. Choose a variable set scope:
+   - **Apply to all workspaces in this organization:** Terraform Cloud will automatically apply this global variable set to all existing and future workspaces.
+   - **Apply to specific workspaces:** Use the text field to search for and select one or more workspaces where Terraform Cloud should apply this variable set.
+
```

Comment:
should we add a warning that users can add variable sets from their workspace?

"Apply to specific workspaces" doesn't prohibit other workspaces from accessing and using the variable set

Reply:
YES good call - I'll push up a fix for this.